### DISCUSSION TITLE
Bathing Env not connecting anymore

### DISCUSSION THREAD

**Discussion by amagasso** [empriselab/RCareWorld#155] [Help - Programming] [answered]
Hello organizers,
It seems that the new version of the bathing env (with the person random seed) is not working.  When launching the "test_bathing.py" file, the program get stuck at "Waiting for connection on port: XXXX".  This is only for the bathing environment, the dressing environment, works fine.

**Comment by YoruCathy** [comment, marked as answer]
Hi, make sure you are on the latest commit of `phy-robo-care`. Also, make sure you did `git lfs pull` to download the large files.

**Reply by amagasso** [reply]
Ok thanks. git lfs pull did the trick. I guess I was missing  some files.
